src/serverrpc.cpp

```diff
@@ -81,18 +92,29 @@ CServerRpc::CServerRpc ( CServer* pServer, CRpcServer* pRpcServer, QObject* pare
             {
                 continue;
             }
+
             QJsonObject client{
                 { "id", i },
                 { "address", vecHostAddresses[i].toString ( CHostAddress::SM_IP_PORT ) },
                 { "name", vecsName[i] },
                 { "jitterBufferSize", veciJitBufNumFrames[i] },
                 { "channels", pServer->GetClientNumAudioChannels ( i ) },
+                { "instrumentCode", vecChanInfo[i].iInstrument },
+                { "instrumentName", CInstPictures::GetName ( vecChanInfo[i].iInstrument ) },
+                { "city", vecChanInfo[i].strCity },
+                { "countryCode", vecChanInfo[i].eCountry },
```

Comment:
Do we have a specific use case for exposing the numeric country code?
Unless there is a very good reason, we should not expose it here. IIRC the country code is Qt-native here. This means, the interpretation of this code would be influenced by whether a server is running a Qt5 or a Qt6 build.

If we do want to expose something besides the country name, we should expose the ISO code, I think.

Reply:
@Rob-NY This is still outstanding I think

Reply:
@Rob-NY ?

Reply:
> If we do want to expose something besides the country name, we should expose the ISO code, I think.


This would probably be https://doc.qt.io/qt-6.2/qlocale.html#codeToScript or https://doc.qt.io/qt-6.2/qlocale.html#bcp47Name or https://doc.qt.io/qt-5/qlocale.html#name then.

Reply:
After further thinking, probably the countryName is sufficient.

Reply:
The issue here is the QT5/QT6 change and inconsistency.